reactioncommerce/reaction
# Fix 3541 - remove "admin" permission from shop manager role

### Fix for #3541 
### To test
- Start the app on this branch
- Notice the migrations will run in your terminal. Something like(Running up() on version 19)
- Invite a user to Shop Manager group
- Remove `email` permission from Shop Manager role
- Login as the invited user, you shouldn’t be able to see the email icon in the sidebar.
- Now invite a second user to the Shop Manager role.
- Remove `i18n` permission from the Shop Manager group.
- Login as the invited user in incognito mode.
- Observe that i18n menu is not available to you.
